### ISSUE TITLE
Copyright year not updated

### ISSUE THREAD

**Issue by KucharczykL** [NginxProxyManager/nginx-proxy-manager#1052]
https://github.com/jc21/nginx-proxy-manager/blob/b42cc9ed3ef3a875feae5026c8f7b8b71b7856e9/frontend/js/i18n/messages.json#L63

I thought I was on old version because the year is still 2019 in the footer. This whole line should probably not be in the translatable strings because there is nothing to translate and it will inevitably get obsolete (as evidenced by this issue).

**Comment by jakern** [comment]
I have heard that it shouldn't be updated https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

**Comment by jc21** [comment]
Yep and I was aware somehow that copyrights only last 5 years - though I can't back that statement up atm 🤷 

As someone in the stack overflow commented, it could just be removed.

**Comment by johntdavis84** [comment]
**_Disclaimer: I am a lawyer, but this is not legal advice._**

For the United States, re: Copyright Term: https://www.copyright.gov/help/faq/faq-duration.html

> **How long does a copyright last?**
> The term of copyright for a particular work depends on several factors, including whether it has been published, and, if so, the date of first publication. As a general rule, for works created after January 1, 1978, copyright protection lasts for the life of the author plus an additional 70 years. For an anonymous work, a pseudonymous work, or a work made for hire, the copyright endures for a term of 95 years from the year of its first publication or a term of 120 years from the year of its creation, whichever expires first. For works first published prior to 1978, the term will vary depending on several factors. To determine the length of copyright protection for a particular work, consult chapter 3 of the Copyright Act (title 17 of the United States Code). More information on the term of copyright can be found in Circular 15a, Duration of Copyright, and Circular 1, Copyright Basics.

>
> **Do I have to renew my copyright?**
> No. Works created on or after January 1, 1978, are not subject to renewal registration. As to works published or registered prior to January 1, 1978, renewal registration is optional after 28 years but does provide certain legal advantages. For information on how to file a renewal application as well as the legal benefit for doing so, see Circular 15, Renewal of Copyright, and Circular 15a, Duration of Copyright.

**Re: Registration and Enforceability (beware massive oversimplification here):** In the United States,
1. You automatically have a copyright in your eligible work form the moment of its creation (for source code, from the moment you fix it in a tangible medium (e.g., save it to disk)).
2. Government registration of copyright (1) allows you to enforce your copyright in federal court; 
3. Ownership of copyright does not depend on marking (that is, you don't have to put the copyright text in the footer to maintain your copyright), though the footer text is useful proof of real public notice of your copyright claim.

As others have mentioned, I would suggest keeping the copyright and making it a date range. Earliest year to current year. If nothing else, that's super useful for seeing how old a project is and if it's still in development (assuming you manually update the year).

**Comment by war59312** [comment]
Indeed, I'd just change to javascript so always the latest year.
